style.css

```diff
@@ -123,6 +168,45 @@ label{
     background-color: white;
 } 
 
+/* Utility classes */
 .no-border{
     border : none;
 }
+.show{
+    animation: crawlIn 1s ease forwards;
+}
+.hide{
+    animation: crawlOut 1s ease forwards;
+}  
+
+@keyframes crawlIn {
+    0%{
+        opacity: 0;
+        top: -0.5rem;
+        left: -0.5rem;
+    } 
+    50%{
+        opacity: 0.5; 
+        top: 8rem;
+        left: 5rem;
+    }
+    100%{
+        opacity: 1;
+        top: 4.5rem;
+        left: 3rem;
+    }
+
+} 
+
+@keyframes crawlOut{
+    0%{
+        opacity: 1; 
+        top: 4.5rem; 
+        left: 8rem;
+    }
+    100%{
+        opacity: 0; 
+        top: 4.5rem; 
+        left: -0.5rem;
```

Comment:
I solved the problem by changing the value of `left: -50rem;`, I meant it like this from the beginning I don't know how it came out as 0.5. 
I can't see the `display: none;` you mentioned, can you say more about it.

Reply:
Yes, it works as you did by increasing your animate to the left side. I meant by `display: none ` is to add this property to the `.hide `class. However, there is no need for adding it now just keep it as you did